contracts/VotingEscrowV2Upgradeable.sol

```diff
@@ -325,24 +350,33 @@ contract VotingEscrowV2Upgradeable is
         // or if the lock is a permanent lock, then _oldLocked.end == 0
         // value == 0 (extend lock) or value > 0 (add to lock or extend lock)
         // newLocked.end > block.timestamp (always)
-        _checkpointLock(_tokenId, _oldLocked, newLocked);
+        /// @dev Prevent DOS by malicious users
+        bool markLockModified = !(depositType == DepositType.CREATE_LOCK_TYPE ||
```

Comment:
I would need more clarity over this. Do you have a reference to the audit issue?

Reply:
Ahhh I see, if someone spams increase lock for another lock that doesn't belong to him, he can prevent the third party from sending it. Correct? 

Though, wouldn't this check defeat the purpose of `markLockModified`

Reply:
Here are the details:

### Audit 1
> Recommendation: Consider implementing a grace period between token splits and
transfers. For example a transfer should not be allowed if a tokenId
was split in the last 60 seconds. The same approach can be used for
the claim and increaseUnlockTime functions.
It might actually make the most sense to implement a nftModified
mapping with the timestamp of the last modification and a check
within the _beforeTokenTransfer function that ensures the tokenId
was not modified recently.

https://github.com/Lynexfi/lynex-contracts/pull/44/commits/06cc076620dbc11bca664bbe5c138693c14c11b2


### Audit 2
> IncreaseAmount() is one such function that increases the vesting
amount of a locked NFT. The issue is that anyone can call it with a
minimum amount of 1 wei, which allows a malicious user to
constantly reactivate the lock and practically DOS the token
transfers at almost no cost.

> Recommendation Consider making the increaseAmount function permissioned with
isAuthorized so that only approved parties or the owner can
actually increase the amount. We do not see any benefit for this
function to be permissionless.

https://github.com/Lynexfi/lynex-contracts/pull/65/commits/d04618dd9afe63d51e5bbb83aaf811c85f956e7e

### Resources
https://drive.google.com/drive/u/0/folders/1IQEOj1yGU6qzPnaOeLHmkBDRgp54cynU